defaults/main/login-window.yml

```diff
@@ -0,0 +1,9 @@
+---
+LoginWindow_DisableConsoleAccess: no             # Toggle login for hidden user '>Console'
+LoginWindow_GuestEnabled: no                     # Toggle guest login
+LoginWindow_LoginwindowText: ""                  # Text message to show in the login screen
```

Comment:
`LoginWindow_LoginwindowText`shoud be `LoginWindow_LoginWindowText` with uppercase W

Reply:
done